docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -145,12 +160,25 @@ Please make sure to check the troubleshooting guides depending on your weather s
 
 ## Location Not Verified
 
-It seems that your location is different from the one you set while claiming your station.
-Please verify your location through the mobile app (Station details -> Settings -> Edit Location)
+It appears that the location of your weather station is different from the one you had originally set while claiming it. 
+Please verify your location through the mobile app.
+<div class="explanation small-big">
+ <img src={useBaseUrl("/img/project/settings.png")} />
+ <img src={useBaseUrl("/img/project/edit_location.png")} />
+ <img src={useBaseUrl("/img/project/edit_location2.png")} />
+</div>
+
 In detail, the [Proof of Location (PoL) mechanism](/project/proof-of-location) flagged your weather station because the location reported by its GPS receiver(s) is different than the location you declared during the setup and claiming of the station.
 
-This happens because either you moved the station from its initial installation site, or you declared a wrong location during the onboarding of the station on your mobile app.
-Your station will be exempt from reward distribution until the weather station is moved back to the declared location or the location is updated by using the "relocate station" function on your mobile app.
+This could happen because either you have moved your station from its initial installation site, or you had declared a wrong location during the onboarding process of the station on your mobile app. 
+
+
+As a result, your station will not be eligible for reward distribution until it is moved back to the declared location or the location is updated by using the "relocate station" function on your mobile app.
+
+:::tip
+When discussing the weather, we always mention a specific location, either directly or indirectly. Therefore, the data provided by weather stations in the WeatherXM Network is meaningless unless we know the location of each station at all times.
+It is crucial for the stability of the Network to have weather stations that can report their current location and remain stationary.
+:::
 
 :::note
 During the beta phase and while the project is still on a test-net, the Proof of Location penalty imposed to devices that went through a relocation is reduced to 2 days for testing purposes
```

Comment:
this should be moved below right after
`Please wait 7 days and your weather station will start earning rewards again.`
and rephrased
`During the beta phase and while the project is still on a test-net, the Proof of Location penalty imposed to devices that went through a relocation is reduced to 2 days for testing purposes, however this may change once the project moves to main-net`.